tools/hangar/src/__snapshots__/invalid.test.ts.snap

```diff
@@ -137,19 +137,15 @@ Error at ../../../../examples/tests/invalid/mut_container_types.w:25:23 | Expect
 `;
 
 exports[`optional_invoke.w > wing test (--target sim) - invalid > stderr 1`] = `
-"Compilation failed with 5 error(s)
-Error at ../../../../examples/tests/invalid/optional_invoke.w:3:1 | Unknown parser error.
-Error at ../../../../examples/tests/invalid/optional_invoke.w:3:10 | Unknown parser error.
-Error at ../../../../examples/tests/invalid/optional_invoke.w:3:21 | Expected expression
-Error at ../../../../examples/tests/invalid/optional_invoke.w:7:1 | Unknown parser error.
-Error at ../../../../examples/tests/invalid/optional_invoke.w:8:1 | Unknown symbol \\"f\\""
+"Compilation failed with 2 error(s)
+Error at ../../../../examples/tests/invalid/optional_invoke.w:7:1 | Cannot call into inflight phase while preflight
+Error at ../../../../examples/tests/invalid/optional_invoke.w:8:1 | Cannot call into inflight phase while preflight"
 `;
 
 exports[`optionals.w > wing test (--target sim) - invalid > stderr 1`] = `
-"Compilation failed with 3 error(s)
-Error at ../../../../examples/tests/invalid/optionals.w:1:1 | Unknown parser error.
-Error at ../../../../examples/tests/invalid/optionals.w:1:10 | Unknown parser error.
-Error at ../../../../examples/tests/invalid/optionals.w:6:1 | Unknown symbol \\"f\\""
+"Compilation failed with 2 error(s)
+Error at ../../../../examples/tests/invalid/optionals.w:6:1 | Cannot call into inflight phase while preflight
```

Comment:
Something's wrong in this test, it's not producing anything related to it's name: "optionals". We need to fix it, that's why err snapshots are important.

Reply:
tweaked it